neqo-http3/src/control_stream_local.rs

```diff
@@ -39,8 +43,35 @@ impl ControlStreamLocal {
     }
 
     /// Send control data if available.
-    pub fn send(&mut self, conn: &mut Connection) -> Res<()> {
+    pub fn send(
+        &mut self,
+        conn: &mut Connection,
+        recv_conn: &mut HashMap<u64, Box<dyn RecvStream>>,
```

Comment:
control_stream now depends on the recv_conn attribute of `Http3Connection`

Reply:
It is ok. I think this is a good approach.